tests/spec_decode/utils.py

```diff
@@ -50,8 +50,8 @@ def new_execute_model(*args, **kwargs):
 
 
 def zero_kv_cache(cache_engine: CacheEngine):
-    assert cache_engine.gpu_cache
-    for key_blocks, value_blocks in cache_engine.gpu_cache:
+    assert cache_engine[0].gpu_cache
```

Comment:
This seems either wrong code or wrong typing

Reply:
Should be fixed now!